gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -227,16 +227,16 @@
               <label translate>Folder Type</label>
               &nbsp;<a href="{{docsURL('users/foldertypes')}}" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
               <select class="form-control" ng-change="setDefaultsForFolderType()" ng-model="currentFolder.type" ng-disabled="editingFolderExisting() && currentFolder.type == 'receiveencrypted'">
-                <option value="sendreceive" translate>Send &amp; Receive</option>
-                <option value="sendonly" translate>Send Only</option>
-                <option value="receiveonly" translate>Receive Only</option>
-                <option value="receiveencrypted" ng-disabled="editingFolderExisting()" translate>Receive Encrypted</option>
+                <option value="sendreceive">{{friendlyNameFromFolderType('sendreceive').replace('&amp;amp;', '&amp;')}}</option>
+                <option value="sendonly">{{friendlyNameFromFolderType('sendonly')}}</option>
+                <option value="receiveonly">{{friendlyNameFromFolderType('receiveonly')}}</option>
+                <option value="receiveencrypted" ng-disabled="editingFolderExisting()">{{friendlyNameFromFolderType('receiveencrypted')}}</option>
```

Comment:
Could probably simplify this whole block to use `currentFolder.type` directly.  As in the other place.